docs/changelogs/v4.1.0.md

```diff
@@ -0,0 +1,239 @@
+**Feature or improvement description**
+Pull request to merge `rc-4.1.0` into `main` and create a tagged release for v4.1.0
+
+See the milestone and project pages for additional information
+
+    https://github.com/OpenFAST/openfast/milestone/17
+
+Test results, if applicable
+See GitHub Actions
+
+### Release checklist:
+- [ ] Update the documentation version in docs/conf.py
+- [ ] Update the versions in docs/source/user/api\_change.rst
+- [ ] Update version info in openfast\_io/pyproject.toml (`openfast_io` package)
+- [ ] Update version info in glue-codes/python/pyproject.toml (`pyOpenFAST` package for testing)
+- [ ] Verify readthedocs builds correctly
+- [ ] Create an annotated tag in OpenFAST during merge (mark as most recent if necessary)
+- [ ] Create a merge commit in r-test and add a corresponding annotated tag
+- [ ] Upload Docker image
+- [ ] Compile executables for Windows builds
+    - [ ] `AeroDisk_Driver_x64.exe`
+    - [ ] `AeroDyn_Driver_x64.exe`
+    - [ ] `AeroDyn_Driver_x64_OpenMP.exe`
+    - [ ] `AeroDyn_Inflow_c_binding_x64.dll`
+    - [ ] `AeroDyn_Inflow_c_binding_x64_OpenMP.dll`
+    - [ ] `BeamDyn_Driver_x64.exe`
+    - [ ] `DISCON.dll (x64)`
+    - [ ] `DISCON_ITIBarge.dll (x64)`
+    - [ ] `DISCON_OC3Hywind.dll (x64)`
+    - [ ] `DISCON_SC.dll (x64)`
+    - [ ] `FAST.Farm_x64.exe`
+    - [ ] `FAST.Farm_x64_OMP.exe`
+    - [ ] `FAST_SFunc.mexw64`
+    - [ ] `HydroDynDriver_x64.exe`
+    - [ ] `HydroDyn_C_Binding_x64.dll`
+    - [ ] `IinflowWind_c_binding_x64.dll`
+    - [ ] `InflowWind_Driver_x64.exe`
+    - [ ] `InflowWind_Driver_x64_OpenMP.exe`
+    - [ ] `MoorDyn_Driver_x64.exe`
+    - [ ] `MoorDyn_c_binding_x64.dll`
+    - [ ] `OpenFAST-Simulink_x64.dll`
+    - [ ] `openfast_x64.exe`
+    - [ ] `SeaStateDriver_x64.exe`
+    - [ ] `SeaState_c_binding_x64.dll`
+    - [ ] `SimpleElastoDyn_x64.exe`
+    - [ ] `SubDyn_x64.exe`
+    - [ ] `Turbsim_x64.exe`
+    - [ ] `UnsteadyAero_x64.exe`
+
+# Changelog (from 4.0.5)
+
+## Overview
+
+This release removes the SuperController from _FAST.Farm_ and adds several important new features including direct coupling of _MoorDyn_ to _SeaState_, rectangular members in _HydroDyn_ and _SubDyn_, and a new _pyOpenFAST_ module for calling c-binding libraries directly from Python.
```

Comment:
Also suggest mentioning here the new vortex-induced vibration (VIV) capability in MoorDyn.

Reply:
That was an oversight.  It is now added.